cypress/integration/individual-pages/checkYourAnswers.spec.ts

```diff
@@ -1,16 +1,40 @@
+import { givenIHaveFilledInCheckBeaconDetailsPage } from "../common/i-can-enter-beacon-information.spec";
 import {
+  andIAmAt,
   givenIHaveACookieSetAndIVisit,
+  givenIHaveClicked,
   iCanSeeAPageHeadingThatContains,
+  thenTheInputShouldOnlyContain,
 } from "../common/selectors-and-assertions.spec";
 
 describe("As a beacon owner, I want to check the details that were submitted", () => {
-  const pageUrl = "/register-a-beacon/check-your-answers";
+  const checkYourAnswerspageUrl = "/register-a-beacon/check-your-answers";
+  const beaconInformationPageUrl = "/register-a-beacon/beacon-information";
+  const applicationCompletePageUrl = "/register-a-beacon/application-complete";
 
-  beforeEach(() => {
-    givenIHaveACookieSetAndIVisit(pageUrl);
-  });
+  const acceptAndSendButtonSelector = ".govuk-button--start";
+  const homePageLinkSelector = ".govuk-header__link--homepage";
+  const startButtonSelector = ".govuk-button--start";
+  const manufacturerFieldSelector = "#manufacturer";
+  const modelFieldSelector = "#model";
+  const hexIdFieldSelector = "#hexId";
 
   it("should display the page title", () => {
+    givenIHaveACookieSetAndIVisit(checkYourAnswerspageUrl);
     iCanSeeAPageHeadingThatContains("Check your answers");
   });
+
+  it("should clear the form when I click Accept and Send", () => {
+    givenIHaveACookieSetAndIVisit(beaconInformationPageUrl);
+    givenIHaveFilledInCheckBeaconDetailsPage();
+    andIAmAt(checkYourAnswerspageUrl);
+    givenIHaveClicked(acceptAndSendButtonSelector);
+    andIAmAt(applicationCompletePageUrl);
+    givenIHaveClicked(homePageLinkSelector);
+    andIAmAt("/");
+    givenIHaveClicked(startButtonSelector);
+    thenTheInputShouldOnlyContain("", manufacturerFieldSelector);
+    thenTheInputShouldOnlyContain("", modelFieldSelector);
+    thenTheInputShouldOnlyContain("", hexIdFieldSelector);
+  });
```

Comment:
<3 seeing decent Cypress testing!

@matthew-a-carr @NyWeb @stuart-madetech FYI How do you think this test would work when the cached draft registration is stored elsewhere (e.g. DynamoDB or a table in the beacon registration service DB?)

I reckon we still merge as-is and look at the Cypress test suite as a whole in the coming weeks.